src/components/Transparency/MonthlyTotal.css

```diff
@@ -3,6 +3,24 @@
   display: flex;
 }
 
+.MonthlyTotal--Full {
+  max-height: 1500px;
+  -webkit-transition: max-height 0.8s;
+  -moz-transition: max-height 0.8s;
+  transition: max-height 0.8s;
+}
+
+.MonthlyTotal--Overview {
+  overflow: hidden;
+  -webkit-transition: max-height 0.3s;
+  -moz-transition: max-height 0.3s;
+  transition: max-height 0.3s;
+}
+
+.MonthlyTotal.MonthlyTotal--Overview {
+  max-height: 398px;
```

Comment:
Can this propoerty be in the class above `.MonthlyTotal--Overview`?

Reply:
Thanks, I was testing and forgot about it